stix_shifter/stix_transmission/src/modules/splunk/splunk_results_connector.py

```diff
@@ -12,11 +12,12 @@ def create_results_connection(self, search_id, offset, length):
         try:
             response = self.api_client.get_search_results(search_id, offset, length)
             response_code = response.code
-            response_json = json.load(response) 
+            response_json = json.load(response)
+
             if "results" in response_json:
-                results = [{}] if (response_json['results'] == []) else response_json['results']
+                results = [] if (response_json['results'] == []) else response_json['results']
```

Comment:
kinda confusing to me. i thought we already handled that in StixPatternToQuery, right?
And also not sure why do we need this checking here. if the response is non `200` than this checking is useless. if the successful response always returns with `results` json object than we can just set 
`return_obj['data'] = response_json['results']`

Reply:
It is a one line assignment statement.  I never wrote the code I'm just fixing it.  If results is empty we want to return an empty list for our pipeline.   This is result set translation.